### PR TITLE
Resolve real path of tracked komodo release

### PR DESCRIPTION
**Pull request by ManInFez**

### CODE REVIEW

File: .github/workflows/python-app.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ on:
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
```
Comment (sondreso) [inline .github/workflows/python-app.yml:15]: Why do we have to specify this? :thinking:

Comment (ManInFez) [reply .github/workflows/python-app.yml:15]: ubuntu-latest is now 20.04 and there python 3.6 is not shipped. When running apt-get install python 3.6 it does not put the executable in the expected /usr/bin folder :shrug:

Comment (jondequinor) [reply .github/workflows/python-app.yml:15]: But it should still be on PATH right? It's probably being installed in usr/local but that is ok

Comment (ManInFez) [reply .github/workflows/python-app.yml:15]: Yes, it ends up somewhere. This is just minimal change for the bug-fix.

**Review by sondreso — APPROVED** [review]
LGTM, a separate issue will be created for the discussion [here](https://github.com/equinor/komodoenv/pull/32/files#r682312731).

**Comment by ManInFez** [comment]
https://github.com/equinor/komodoenv/issues/33

**Pull request merged**